test_runner/src/test/kotlin/ftl/cli/firebase/test/android/AndroidRunCommandTest.kt

```diff
@@ -74,4 +75,13 @@ class AndroidRunCommandTest {
         CommandLine(cmd).parse("--test", "myTestApp.apk")
         assertThat(cmd.test).isEqualTo("myTestApp.apk")
     }
+
+    @Test
+    fun testTargets_parse() {
+        val cmd = AndroidRunCommand()
+        CommandLine(cmd).parse("--test-targets", "class com.foo.Clazz", "--test-targets", "package com.my.package")
```

Comment:
probably better to extract `"class com.foo.Clazz", "--test-targets", "package com.my.package"` to a variable.

I think you can make them a list and then destructure with *.

val expectedTargets = listOf("class com.foo.Clazz", "--test-targets", "package com.my.package")
...